Adding cohen2020causal

lsbuschoff opened the pull request:
Adding data for the 2020 Cohen et al. paper "The rational use of causal inference to guide reinforcement learning strengthens with age".

marcelbinz commented:
looking good. A few things before merging:

*  make sure RTs are in ms.
* The length of the RTs list should match the number of choices. At present, it is only half of it (probably either the bandit task or the intervention question missing). If one of them is not available, please fill with nan. Each item in the list should match the corresponding choice in the prompt.

lsbuschoff commented:
I fixed the RTs to be in ms. The subjects make two decisions each trial, first they decide which mine to choose (and for this there are RTs recorded) and then whether they think the reward they saw was due to intervention by the agent (for this there is no RT recorded). I changed the structure of the RTs to reflect this so there is now always a RT in ms for the choice question and a nan entry for the intervention question.

marcelbinz commented:
Thanks again for the contribution!
